avidemux/winInstaller/ChangeLog.release

```diff
@@ -8,92 +8,62 @@
+      <li>Blur (on selected area)</li>
+      <li>Cartoon</li>
+      <li>Color balance (adjust shadow, midtone and highlight)</li>
+      <li>Chroma hold (monochrome effect with kept colors)</li>
+      <li>Chroma key (replace "green screen" with an image)</li>
+      <li>DelogoHQ (clean up arbitrary shaped logo)</li>
+      <li>Dynamic threshold</li>
+      <li>Fit to size (resize and pad to the specified size)</li>
+      <li>Flip (vertically/horizontally flip the image)</li>
+      <li>Grid (video wall effect)</li>
+      <li>Image stabilizer (reduce camera shakiness)</li>
+      <li>Luma stabilizer (reduce flickering)</li>
+      <li>Mirror</li>
+      <li>Posterize</li>
+      <li>Quadrilateral transformation (four point transform)</li>
+      <li>Wavelet denoiser</li>
+      <li>Wavelet sharpener</li>
+    </ul>
 
     <h2>New Features</h2>
 
     <ul>
-      <li>Many new video filters added by courtesy of
-          <a href="https://github.com/szlldm">szlldm</a>.</li>
-      <li>A dedicated MOV muxer is now available, replacing silent fallback to
-          MOV muxing mode within the MP4 muxer.</li>
-      <li>A subset of color info relevant for HDR support is retained in copy
-          mode when both input and output video are stored in Matroska / WebM
-          containers. Settings to add or override color info have been added to
-          MKV and WebM muxers.</li>
-      <li>The indexer in the MPEG-TS demuxer now detects resolution changes in
-          MPEG-2 and H.264 streams which are entirely unsupported and suggests
-          aborting indexing. If the user chooses to continue, it records file
-          offsets of all resolution changes to the log. This allows advanced
-          users to split the file at these offsets using other tools like
-          <em>head</em> and <em>tail</em> into chunks which can be processed
-          with Avidemux.</li>
-      <li>YUV varieties of the lossless Ut Video codec are supported now for
-          decoding and encoding via bundled FFmpeg libraries.</li>
-      <li>Uncompressed 24-bit audio is now supported for a subset of file formats.</li>
-      <li>Limited support for 7.1 channel layout.</li>
-      <li>Decode and stream copy support for XLL substream in DTS audio tracks.</li>
-      <li>Audio properties are now updated on resuming playback. This allows to
-          start saving after an audio format change. When audio format changes
-          during playback, the FFmpeg-based audio decoder will output silence
-          and keep format rather than producing garbage.</li>
-      <li>An option to keep the identity of markers A and B instead of swapping
-          them automatically has been added to Preferences.</li>
-      <li>The keyboard shortcut for "Delete" action has been made customizable
-          with alternative keyboard shortcuts enabled.</li>
+      <li>Resample FPS filter has blend and motion interpolation options.</li>
+      <li>FFV1 encoder is available again.</li>
```

Comment:
"available again" – just curious, when was it available and when disabled/removed?

Reply:
It was available pre 2.6.
https://avidemux.org/smif/index.php?topic=11888.0
https://en.wikipedia.org/wiki/FFV1